yb-voyager/cmd/importData.go

```diff
@@ -1547,3 +1553,72 @@ func cleanStoredErrors(errorHandler importdata.ImportDataErrorHandler, tasks []*
 	}
 	return nil
 }
+
+func BuildCallhomeYBClusterMetrics() (callhome.YBClusterMetrics, error) {
+	yb, ok := tdb.(*tgtdb.TargetYugabyteDB)
+	if !ok {
+		utils.ErrExit("importData: expected tdb to be of type TargetYugabyteDB, got: %T", tdb)
+	}
+
+	clusterMetrics, err := yb.GetClusterMetrics()
+	if err != nil {
+		return callhome.YBClusterMetrics{}, err
+	}
+
+	now := time.Now().UTC()
+	nodes := make([]callhome.NodeMetric, 0)
+	var totalCpuPct, totalMemPct float64
+	for _, nodeMetrics := range clusterMetrics {
+		// in case of err value will be -1
+		cpuPct, err := nodeMetrics.CPUPercent()
+		if err != nil {
+			log.Warnf("callhome: error getting CPU percent for node %s: %v", nodeMetrics.UUID, err)
```

Comment:
return err? 
check the other instances in this function below as well.

Reply:
In this case, i was thinking of just ignoring the error here.
For eg: if parsing to int or something fails for a particular metric of one of the nodes then we can still try for other nodes in cluster.


Although this is not likely to happen, i can return in that case. what do you think?